cpu/efr32mg1p/periph/Makefile

```diff
@@ -0,0 +1,3 @@
+MODULE = periph
+
+include $(RIOTBASE)/Makefile.base
```

Comment:
probably best to wait for #7241 and adapt before merging.

Reply:
Not yet included, but will do once merged.